addons/website/static/tests/tours/edit_link_popover.js

```diff
@@ -113,5 +113,55 @@ tour.register('edit_link_popover', {
         trigger: '.js_add_menu',
         run: function () {}, // it's a check
     },
+    {
+        content: "Close modal",
+        trigger: '.modal-footer .btn-secondary',
+    },
+    // 3. Test other links (CTA in navbar & links in footer)
+    {
+        content: "Click CTA in navbar",
+        trigger: '#top_menu_container a.btn-primary[href="/contactus"]',
+    },
+    {
+        content: "Popover should be shown (3)",
+        trigger: '.o_edit_menu_popover .o_we_url_link:contains("Contact Us")',
+        run: function () {}, // it's a check
+    },
+    {
+        content: "Click 'Home' link in footer",
+        trigger: 'footer a[href="/"]',
+    },
+    {
+        content: "Popover should be shown (4)",
+        trigger: '.o_edit_menu_popover .o_we_url_link:contains("Home")',
+        run: function () {}, // it's a check
+    },
+    // 4. Popover should close when clicking non-link element
+    {
+        content: "Ensure popover is closed",
+        trigger: 'footer h5',
+    },
+    // 5. Double click shouldn't do anything
+    {
+        content: "Double click on link",
+        trigger: 'html:not(:has(.o_edit_menu_popover))', // popover should be closed
+        run: function () {
+            function createRange(el) {
+                var range = document.createRange();
+                range.selectNodeContents(el);
+                var sel = window.getSelection();
+                sel.removeAllRanges();
+                sel.addRange(range);
+            }
+            const $footerHomeLink = $('footer a[href="/"]').first();
+            createRange($footerHomeLink[0]);
+            $footerHomeLink.click().dblclick();
+        },
+    },
+    {
+        content: "Ensure nothing happened on double click (except showing popover but this can't be tested as clicks in tests are done without mouseovering)",
```

Comment:
clicks in tests are done with "mouseovering" ;)

Reply:
Actually, this was written previously of the `range` workaround of previous step, so removed that part of the sentence and actually tested the popover is open.
Double click was not opening the popover without that